Updated Components Styling sections (#532)
IgniteUI/indigo-design-docfx: en/components/switch.md

## Styling

> The Switch comes with styling flexibility through the available options, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.

Reviewer: ```suggestion
The Switch comes with styling flexibility through the available options for the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page or artboard in the Indigo.Design library.
```
